be/tool/main.c

```diff
@@ -32,12 +32,29 @@
 
 #include <stdio.h>              /* printf */
 #include <stdlib.h>             /* EXIT_FAILURE */
+#include <string.h>     	/* strcmp */
```

Comment:
alignment of /* strcmp */

Reply:
its correctly aligned. It is shown properly in vim, some issue with displaying here.